schema/profile.js

```diff
@@ -382,6 +383,47 @@ class Profile extends BaseModel {
       .then(([filters, total, profiles]) => ({ filters, total, profiles }));
   }
 
+  static getFormerAsruProfiles(params) {
+    const { search, limit, offset, sort = {} } = params;
+
+    const asruUserAdded = Changelog.query()
+      .select('modelId')
+      .where({ modelType: 'profile', action: 'update' })
+      .whereJsonSupersetOf('state', { asruUser: true });
+
+    const asruUserRemoved = Changelog.query()
+      .select(this.raw('uuid(model_id) AS id'))
+      .select('updatedAt AS removedAt')
+      .whereJsonSupersetOf('state', { asruUser: false })
+      .whereIn('modelId', asruUserAdded)
+      .as('asruRemoved');
+
+    const query = this.query().innerJoin(asruUserRemoved, 'profiles.id', 'asruRemoved.id');
```

Comment:
I think we want an extra `.where({ asruUser: false })` as well to filter out anyone who was removed and then re-added again.

Reply:
Done. Had this originally and lost it when I moved stuff around, doh.